### ISSUE TITLE
Weekly trade view shows less data than other time views

### ISSUE THREAD

**Issue by boner-cmd** [btdex/btdex#159]
Version 0.4.8

Every other timescale has a bar per unit, with the time scale in units. But weekly shows the timescale in **_days_** instead of weeks. For example, expected results:

One bar per day, timescale in days (TRT)
![image](https://user-images.githubusercontent.com/54084132/118725867-1f4e6000-b7f6-11eb-8f6a-691b20add35a.png)

One bar per hour, timescale in hours (BTC)
![image](https://user-images.githubusercontent.com/54084132/118725959-40af4c00-b7f6-11eb-924d-b08539b9070e.png)

Actual result for weekly:

One bar per week, timescale in days (TRT)
![image](https://user-images.githubusercontent.com/54084132/118726070-6a687300-b7f6-11eb-9222-16153f2e2cb0.png)

This makes it a lot harder to see trends because instead of getting the standard number of bars, you only get four.

**Comment by boner-cmd** [comment]
tl;dr the weeks grid scale should be two weeks, not two days

**Comment by boner-cmd** [comment]
Inconsistent reproduction. Not seeing it on every chart. Unsure if it is per pair or caused by something else.

BNB exhibiting the bug
![image](https://user-images.githubusercontent.com/54084132/118808858-8c520c00-b86f-11eb-84a6-49ea891e787d.png)

DOGE showing the correct timescale
![image](https://user-images.githubusercontent.com/54084132/118808963-a855ad80-b86f-11eb-976c-9e8072a68496.png)

Some of these are even wrong in different ways:

XMR showing 14 APR - 12 MAY
![image](https://user-images.githubusercontent.com/54084132/118809130-d4712e80-b86f-11eb-8a83-fd7a4d2a6cf4.png)

XCH showing 28 APR - 12 MAY
![image](https://user-images.githubusercontent.com/54084132/118809209-ef43a300-b86f-11eb-80ad-27b75adb5a2d.png)

Also wait it's 19 May today what the hell?
